### ISSUE TITLE
try sen with PyPy

### ISSUE THREAD

**Issue by TomasTomecek** [TomasTomecek/sen#56]
this may be even worth making sen compatible with python 2 since atm pypy3 supports only python 3.2; it really depends on performance speedup (most of the time, sen is waiting for response from docker anyway)

**Comment by funkyfuture** [comment]
some 2 cents:

- you may have read the news that mozilla is funding 3.5 support of PyPy
- no, it's not worth the effort to make an end-user-application compatible with python 2
- docker-py is pypy-compatible, though the maintainers [quarrel to acknowledge](https://github.com/docker/docker-py/pull/749)

**Comment by TomasTomecek** [comment]
Yup, I'm supper excited about mozilla donating PyPy to support 3.5. It's true that most of the time sen is waiting for docker engine to respond, but on the other hand, it also spends a lot of time preparing views (=processing text). I feel like that PyPy could really speed that up. And then it could be even possible to do some insane real-time features.

Good job with the docker-py PR. Pretty sad that it didn't get merged.
